include/rppt_tensor_morphological_operations.h

```diff
@@ -59,7 +59,7 @@ extern "C" {
  * \param [in] kernelSize kernel size for box filter (a single Rpp32u odd number with kernelSize = 3/5/7/9 that applies to all images in the batch)
```

Comment:
combine both functions in one #ifdef block

Reply:
same comment as above